core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -162,6 +162,39 @@ public static String get(PropertyKey key) {
     return lookup(rawValue);
   }
 
+  /**
+   * Gets the display value for the given key in the {@link Properties}; Credentials will be masked.
+   * If this key is not found, a RuntimeException is thrown.
+   *
+   * @param key the key to get the value for
+   * @return the display value for the given key
+   */
+  public static String getDisplayValue(PropertyKey key) {
+    String value = get(key);
+    return getDisplayValue(key, value);
+  }
+
+  /**
+   * Gets the display value for the given key and value in the {@link Properties}.
+   * Credentials will be masked.
+   *
+   * @param key the key to get the value for
+   * @param value value of the property
+   * @return the display value for the given key
+   */
+  public static String getDisplayValue(PropertyKey key, String value) {
```

Comment:
is this method necessary? technically, it is only used in `toRawMapInternal`

Reply:
It's also used in the method `Configuration.getDisplayValue(PropertyKey key)`

Reply:
sure. my question is do we really need `getDisplayValue(PropertyKey key, String value)`? this method is a bit wired to me as it takes the value as the input too. Since it is only used in `toRawMapInternal`, can we just make special handle there?

Reply:
Per our discussion, unified method into `get(PropertyKey, ValueOptions)`